corehq/apps/reports/standard/cases/utils.py

```diff
@@ -177,11 +175,9 @@ def get_case_owners(can_access_all_locations, domain, mobile_user_and_group_slug
         location_owner_ids,
         assigned_user_ids_at_selected_locations,
     ))
-    if settings.IS_SAAS_ENVIRONMENT:
-        # temporary code to understand usages of this function that result in a lot of owner_ids
-        limit = SystemLimit.get_limit_for_key("owner_id_limit", 1000, domain=domain)
-        if len(owner_ids) > limit:
-            notify_exception(None, "Exceeded recommended owner id count", details={"count": len(owner_ids)})
+    limit = SystemLimit.get_limit_for_key(OWNER_ID_LIMIT_KEY, DEFAULT_OWNER_ID_LIMIT, domain=domain)
+    if len(owner_ids) > limit:
+        raise TooManyOwnerIDsError
```

Comment:
Is there already something that catches all errors and displays them to the user?  Otherwise wouldn't this just show up as a 500?

Reply:
Yeah this was an oversight on my part. Thanks for catching!